app/experimenter/templates/experiments/list.html

```diff
@@ -134,16 +137,46 @@ <h5>{{ experiment.population }}</h5>
     </div>
 
     {% for field in filter.form %}
-      {% ifequal field.name "archived" %}
+      {% if field.name == "archived" %}
         <label>
           {{ field }}
           {{ field.label }}
         </label>
+      {% elif field.name == "show_experiment_changes" %}
+        <hr class="my-4">
+        <div class="form-group">
+          {{ field }}
+        </div>
+
+      {% elif field.name == "start_date_range" %}
+        <div id="start-both-date-inputs" class="form-group">
+          <label class="date-range-label">
+            From
+          </label>
+          {{ field }}
+        </div>
+
+      {% elif field.name == "pause_date_range" %}
+        <div id="pause-both-date-inputs" class="form-group">
+          <label class="date-range-label">
+            From
+          </label>
+          {{ field }}
+        </div>
+
+      {% elif field.name == "end_date_range"%}
+        <div id="end-both-date-inputs" class="form-group">
+          <label class="date-range-label">
+            From
+          </label>
```

Comment:
I feel like it might have been better to put the `From` label in the template too (although this is less important if we only have one such field).

Reply:
Made this change!